silkrpc/ethbackend/backend.hpp

```diff
@@ -177,6 +194,20 @@ class BackEnd final {
         co_return count;
     }
 
+    asio::awaitable<ExecutionPayload> engine_get_payload_v1(uint64_t payload_id) {
+        const auto start_time = clock_time::now();
+        EngineGetPayloadV1Awaitable npc_awaitable{executor_, stub_, queue_};
+        ::remote::EngineGetPayloadRequest req;
+        req.set_payloadid(payload_id);
+        const auto reply = co_await npc_awaitable.async_call(req, asio::use_awaitable);
+        co_return decode_execution_payload_from_grpc_format(reply);
+    }
+
+    // just for testing
+    asio::awaitable<::types::ExecutionPayload> execution_payload_to_proto(ExecutionPayload payload) {
```

Comment:
that would require to make the Backend non-final and put all the private functions as `protected`

Reply:
I was unsure about this because declaring private functions as protected, implies it is an a type to be inherited from, which is not thus making it ambigous

Reply:
I also will remove it on a second PR, when i will start working on `engine_newPayload`

Reply:
OK